### PR TITLE
Gui fleet sending v2

### PR DESCRIPTION
**Pull request by spanish-inquisition-4**
Added in client random selecting from three hardcoded fleets and sending it to server.

### CODE REVIEW

File: src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java [hunk 1]
```diff
@@ -94,5 +101,18 @@ private void runTheGame() {
         this.playersLabel.setText("Set Up your ships");
         this.game = new Game();
         this.game.buildPlayersBoard(new BoardController(new GameBoard(this.playersGridPane)));
+        this.fleetSetupButton.setVisible(true);
+    }
+
+    @FXML
+    public void onFleetSetupButtonClicked(){
+        this.game.placePlayersShips();
+        this.sendToServerButton.setVisible(true);
```
Comment (arturAR) [inline src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java]: Why always "this"? Method without parameters and "this", "this", "this"? Defensive programming!

File: src/main/java/com/spanish_inquisition/battleship/server/fleet/Fleet.java [hunk 2]
```diff
@@ -12,4 +12,11 @@ public Fleet(List<Ship> ships) {
     public List<Ship> getShips() {
         return ships;
     }
-}
+
+    @Override
+    public String toString() {
+        return "Fleet{" +
+                "ships=" + ships +
+                '}';
```
Comment (arturAR) [inline src/main/java/com/spanish_inquisition/battleship/server/fleet/Fleet.java]: Why not in one line?

File: src/main/java/com/spanish_inquisition/battleship/server/fleet/Ship.java [hunk 3]
```diff
@@ -12,4 +12,11 @@ public Ship(List<Integer> shipPoints) {
     public List<Integer> getShipPoints() {
         return shipPoints;
     }
-}
+
+    @Override
+    public String toString() {
+        return "Ship{" +
+                "shipPoints=" + shipPoints +
+                '}';
```
Comment (arturAR) [inline src/main/java/com/spanish_inquisition/battleship/server/fleet/Ship.java]: Why not in one line?

File: src/main/java/com/spanish_inquisition/battleship/client/board/BoardController.java [hunk 4]
```diff
@@ -31,4 +36,14 @@ public GameBoard getGameBoard() {
     public Map<Integer, BoardTile> getBoardsIndexTiles() {
         return gameBoard.getIndexTiles();
     }
-}
+
+    public void placeShips() {
+        Map<Integer, BoardTile> indexTiles = gameBoard.getIndexTiles();
+        indexTiles.forEach((integer, boardTile) -> boardTile.setTileStyle(Styles.DEFAULT_TILE_COLOR, Styles.TEXT_BLACK));
```
Comment (mprtcz) [inline src/main/java/com/spanish_inquisition/battleship/client/board/BoardController.java]: 👍

File: src/main/java/com/spanish_inquisition/battleship/client/game/FleetInitializer.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    public FleetInitializer(BoardController boardController) {
+        this.boardController = boardController;
+    }
+
+    public void setUpShips() {
+        boardTiles = boardController.getBoardsIndexTiles();
+        shipPlaces = Fleet.getRandomFleet().getShipPlaces();
+        shipPlaces.forEach(this::changeTileStyle);
```
Comment (mprtcz) [inline src/main/java/com/spanish_inquisition/battleship/client/game/FleetInitializer.java]: Method reference 👍

File: src/main/java/com/spanish_inquisition/battleship/client/network/SocketClient.java [hunk 6]
```diff
@@ -40,8 +40,9 @@ public static SocketClient createSocketClientWithSocket() throws IOException {
 
     public void sendStringToServer(String string) {
         try {
+            logger.log(DEFAULT_LEVEL, "Attempting sent string to server = " + string);
```
Comment (mprtcz) [inline src/main/java/com/spanish_inquisition/battleship/client/network/SocketClient.java]: "Attempting to send..."

**Pull request merged**